frontend/service.go

```diff
@@ -72,7 +72,7 @@ func (s *lwdStreamer) GetLatestBlock(ctx context.Context, placeholder *walletrpc
 	if err != nil {
 		return nil, err
 	}
-	r := &walletrpc.BlockID{Height: uint64(blockChainInfo.Blocks), Hash: []byte(bestBlockHash)}
+	r := &walletrpc.BlockID{Height: uint64(blockChainInfo.Blocks), Hash: parser.Reverse(bestBlockHash)}
```

Comment:
Document this bugfix in the changelog (which looks like it hasn't been updated for a while, but this is an important bugfix for people to be aware of; make any other changes necessary to delineate this fix from the prior changes).

I would also prefer that `bestBlockHash` be in little-endian encoding, i.e. we do not store the hex-decoded-but-not-reversed bytes in a variable (otherwise we run the risk of encountering this exact same bug again in some future refactor). So either move the `parser.Reverse` up to wrap `bestBlockHash` in some way, or inline the `hex.DecodeString` here.